Packs/BmcITSM/Integrations/BmcITSM/BmcITSM.yml

```diff
@@ -2324,6 +2328,364 @@ script:
       name: lastUpdate
     description: Gets the list of incidents that were modified since the last update time. Note that this method is here for debugging purposes. The get-modified-remote-data command is used as part of a Mirroring feature, which is available in Cortex XSOAR from version 6.1.
     name: get-modified-remote-data
+  - name: bmc-itsm-support-group-list
+    description: Lists all support groups. Useful for getting possible (Company, Support Organization, Support Group) triplets.
+    arguments:
+    - name: limit
+      description: The maximum number of records to retrieve.
+      type: String
+      defaultValue: "50"
+      predefined:
+      - ""
```

Comment:
lets get rid of these empty predefined tags (sorry if this was generated by the demistosdk)
it could be that running `demisto-sdk format` will fix issues like these

FYI @YuvHayun